src/Microsoft.ML.AutoML.Interactive/NotebookMonitor.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.DotNet.Interactive;
+using System.Collections.Generic;
+using Microsoft.Data.Analysis;
+using System;
+using System.Threading.Tasks;
+using System.Text.Json;
+
+
+namespace Microsoft.ML.AutoML
+{
+    public class NotebookMonitor : IMonitor
+    {
+        private DisplayedValue _valueToUpdate;
+        private DateTime _lastUpdate = DateTime.MinValue;
+
+        public TrialResult BestTrial { get; set; }
+        public TrialResult MostRecentTrial { get; set; }
+        public TrialSettings ActiveTrial { get; set; }
+        public List<TrialResult> CompletedTrials { get; set; }
+        public DataFrame DataFrame { get; set; }
+
+        public NotebookMonitor()
+        {
+            this.CompletedTrials = new List<TrialResult>();
```

Comment:
>this. [](http://example.com/codeflow?start=12&length=5)

nit: 
do we usually use `this.` in our coding convention? in .NET runtime we usually don't use it except if there is a good reason to do so.

Reply:
Doesn't look like it  (I'm new to this repo). Removed `this.`

Reply:
Nope, coding convention is to not use `this` unless we have to.